spec/Section 6 -- Execution.md

```diff
@@ -350,3 +485,39 @@ cannot be `null` the error is propagated to be dealt with by the parent field.
 
 If all fields from the root of the request to the source of the error return
 `Non-Null` types, then the `"data"` entry in the response should be `null`.
+
+
+### Coercing Field Arguments
+
+Fields may include arguments which are provided to the underlying runtime in
+order to correctly produce a value. These arguments are defined by the field in
+the type system to have a specific input type: Scalars, Enum, Input Object, or
+List or Non-Null wrapped variations of these three.
+
+At each argument position in a query may be a literal value or a variable to be
+provided at runtime.
+
+CoerceArgumentValues(objectType, field, variableValues)
+  * Let {argumentValues} be the argument values provided in {field}.
+  * Let {fieldName} be the name of {field}.
+  * Let {argumentDefinitions} be the arguments defined by {objectType} for the
+    field named {fieldName}.
+  * Let {coercedArgumentValues} be an empty Map.
+  * For each {argumentDefinitions} as {argumentName} and {argumentType}:
+    * If no value was provided in {argumentValues} for the name {argumentName}:
+      * Continue to the next argument definition.
```

Comment:
That's not how things work today, but I agree that's problematic. Filed #224 to follow up with this.